### DISCUSSION TITLE
font size in iOS app (german)

### DISCUSSION THREAD

**Discussion by Squ1dw4rd99** [TomBursch/kitchenowl#457] [Ideas]
could you scale the (german) font size in the iOS app that the "Einkaufsliste" (shopping list) and "Essensplaner" fit in one line? looks really distracting atm (and I don't see the option to change the language to english)![image](https://github.com/TomBursch/kitchenowl/assets/13121941/07d9d962-b199-470c-8a9e-432ee5a8b515)![image](https://github.com/TomBursch/kitchenowl/assets/13121941/01ad3963-205b-40ad-aa92-4ab1fd7f5784)

**Comment by Squ1dw4rd99** [comment]
Addition:
In a recipe when I change the portion count, the german translation of "meal plan" changes (from "Speiseplan" to "Essensplan")
![image](https://github.com/TomBursch/kitchenowl/assets/13121941/d3a8444d-8b34-4dff-9718-f560c54630d7)![image](https://github.com/TomBursch/kitchenowl/assets/13121941/4afc8521-4d26-46a0-b2a3-9b4324b7e79a)

**Comment by TomBursch** [comment]
Hey, thanks for your suggestions. I've added both, and they should be included in the next release 🙂
